models/base.js

```diff
@@ -1,10 +1,13 @@
-const bookshelf = require('../bookshelf');
+const bookshelf = require('../db/bookshelf');
 
 module.exports = bookshelf.Model.extend({
     hasTimestamps: true
 }, {
     findAll: function(filter, options) {
-        return this.forge().where(filter).fetchAll(options);
+        if(filter)
+            return this.forge().where(filter).fetchAll(options);
```

Comment:
Is this if statement necessary?

Reply:
Yes, because if no filter object was passed, the empty where() would throw an exception